packages/block-library/src/columns/block.json

```diff
@@ -4,6 +4,9 @@
 	"attributes": {
 		"verticalAlignment": {
 			"type": "string"
+		},
+		"width": {
+			"type": "number"
```

Comment:
Is it ok to add another attribute like that to pass Columns block width to it's Column ?

Reply:
I've manage to use BlockSettings instead of passing by attributes so it will be deleted in next commit

Reply:
done